### ISSUE TITLE
Grid issue: missing .uk-grid-margin and .uk-first-column in Firefox

### ISSUE THREAD

**Issue by oohnajra** [uikit/uikit#2553]
### UIkit version

3.0.0-beta.18

### Reproduction Link

Unfortunately I've not been able to reproduce the issue with a Codepen.

### Steps to reproduce
Using Firefox 52.0.1 on Windows 10. When the browser window is 1600+ pixels the classes `.uk-grid-margin` and `.uk-first-column` aren't being added to the 2nd row of grid items on initial page load, so there's no margin between the 1st and 2nd row. Strangely enough these classes are being added to 3rd row and up. When manually resizing the browser window, the classes are added. The classes are also added when the browser window is less than 1600 pixels on initial page load. I've been able to reproduce the issue via Browserstack, so it's not caused by a local setting or add-on. The issue doesn't occur in any other browser I've tested, which includes Chrome, IE11 and Edge.

![screenshot-2017-03-27-19-08-13](https://cloud.githubusercontent.com/assets/161342/24369789/6b610242-1325-11e7-89cc-5d435a08bff5.png)

![source](https://cloud.githubusercontent.com/assets/161342/24369790/6b61f576-1325-11e7-8633-f4e876e76f30.png)

### What is Expected?
The classes `.uk-grid-margin` and `.uk-first-column` should be added to all relevant grid items on page load.

### What is actually happening?
See my description under "Steps to reproduce"

**Comment by zzseba78** [comment]
Same here, only in Firefox. It's like a random error, sometimes happens, sometimes not... To me it happens at any screen size.

**Comment by oohnajra** [comment]
@zzseba78 Glad to hear I'm not the only one with this issue. Yes, it's pretty random, so I wasn't able to pinpoint the cause or reproduce it in a Codepen.

**Comment by zzseba78** [comment]
Yes, this happens since they released uikit 3, it happens sometimes, and in a specific grid, i mean, all the grids work great, but in one specific this error happens randomly. Strange.

**Comment by janschoenherr** [comment]
Has this been resolved?

**Comment by oohnajra** [comment]
I noticed the issue has been closed. Does this mean that it has been resolved?

**Comment by janschoenherr** [comment]
Sorry, closed without comment. Yes, I believe the issue has been resolved half a year ago.
